tools/scenario_runner.py

```diff
@@ -257,7 +258,7 @@ def run(
         if stage_prefix is not None:
             open('{}.stage1'.format(stage_prefix), 'a').close()
             log.warning('Done with initialization, waiting to continue...')
-            event = gevent.event.Event()
+            event = RaidenGreenletEvent
```

Comment:
shouldnt this be `RaidenGreenletEvent()`?

Reply:
Argh ffs yes. I will let the tests run first since this is just in the scenario runner though